libkernelflinger/ivshmem.c

```diff
@@ -411,3 +426,13 @@ EFI_STATUS ivshmem_init(void)
 
 	return EFI_SUCCESS;
 }
+
+void ivshmem_rot_interrupt(void)
+{
+	io_write_32((void *)((UINT64)(g_ivshmem_dev.bar0_addr + DOORBELL_OFF)), 0x1);
```

Comment:
please define a macro for the interrupt num, 0x1, 0x2

Reply:
done